core/src/main/java/org/openedx/core/data/model/CourseStructureModel.kt

```diff
@@ -108,7 +112,7 @@ data class CourseStructureModel(
         override fun deserialize(
             json: JsonElement?,
             typeOfT: Type?,
-            context: JsonDeserializationContext?
+            context: JsonDeserializationContext?,
```

Comment:
unnecessary code change, need to revert same for the other files.

Reply:
Removed all unnecessary changes.